src/comps/core/Form.tsx

```diff
@@ -14,6 +14,7 @@ type FlowFormProps = {
   children?: React.ReactNode;
   validationSchema?: YupSchema;
   disabled?: boolean;
+  requiredConnection?: "stx" | "btc" | "both";
```

Comment:
qq: do we expect to have a flow requiring both at some time, or this is just a _why not_ thing?

Reply:
a whynot thing